src/extension/background-script/actions/nostr/signEventOrPrompt.ts

```diff
@@ -17,15 +23,60 @@ const signEventOrPrompt = async (message: MessageSignEvent) => {
     };
   }
 
+  const enableMessage: MessageAllowanceEnable = {
+    origin: message.origin,
+    args: {
+      host: message.origin.host,
+    },
+    action: "public/webln/enable",
+  };
+
+  await enable(enableMessage);
+
+  const dbPermissions = await db.permissions
+    .where("host")
+    .equalsIgnoreCase(message.origin.host)
+    .toArray();
+
+  const hasPermission = dbPermissions
+    .filter((permission) => permission.enabled)
+    .map(({ method }) => method)
```

Comment:
I think we should use the dexy API for this. there should be a method to query for entries where enabled=true and method=x

Reply:
Not such good API, but adjusted to what I could find, i.e.:
- https://stackoverflow.com/questions/35679590/dexiejs-indexeddb-chain-multiple-where-clauses
- https://groups.google.com/g/dexiejs/c/G3_W5PssCGA

Reply:
you link to a post that is 7 years ago, is that still relevant? 

this looks pretty much what we need, isn't it? 
https://dexie.org/docs/Table/Table.where()#samples

Reply:
Please check current code, I already adjusted it